docs/api-explorer.yml

```diff
@@ -74,6 +74,35 @@ paths:
           $ref: "#/components/responses/NoResults"
         "500":
           $ref: "#/components/responses/InternalError"
+  /api/explorer/v2/milestones:
+    get:
+      tags:
+        - balance
+      summary: Returns an address's balance.
+      description: >-
+        Returns the balance of IOTA tokens owned by a given bech32 address.
+      parameters:
+        - $ref: "#/components/parameters/startTimestamp"
+        - $ref: "#/components/parameters/endTimestamp"
+        - $ref: "#/components/parameters/sort"
+        - $ref: "#/components/parameters/pageSize"
+        - $ref: "#/components/parameters/cursor"
+      responses:
+        "200":
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BalanceResponse"
+              examples:
+                default:
+                  $ref: "#/components/examples/balance-example"
```

Comment:
Update summary and description and response refs with `MilestonesResponse`

Reply:
Wow, clearly I was very tired when I made these changes.